### ISSUE TITLE
This is weird AF

### ISSUE THREAD

**Issue by katastrophe92** [Sycnex/Windows10Debloater#543]
![hmmmmm](https://user-images.githubusercontent.com/103950507/207536801-58cd0fcf-4b23-43a9-8c31-b2a141a6801d.PNG)
![hmm](https://user-images.githubusercontent.com/103950507/207536786-296573eb-2e38-47fd-8d9e-cf91b27ee278.PNG)

**Comment by katastrophe92** [comment]
I do not know what to think. After a fresh win10 install i just wanted to try new debloat scripts/apps and downloaded a couple new ones. First I ran the classic ShutUP10 and after I just wanted to see how/what the other ones do.
Windows10Debloater I opened first and made a custom list and closed it. That was it. After I opened SophiApp and showed me this message. How does it know in the first place and why does it cares. Did Windows10Debloater made a registry entry or placed a file somewhere and SophiApp is searching for it? Are this 2 apps are not compatible, fighting each other or the developers of this apps do not each other?
It is funny but at the same time I'm confused and do not know what to do now. I deleted both programs and after downloaded just SophiApp and run it and still it is popping the same message. Where are still files or trails of Windows10Debloater on my PC?

**Comment by Sycnex** [comment]
I’m not familiar with the SophiApp. What app is that? It’s more likely that the app you’re using that runs the script did something. Does windows still work?Technically, running any of the windows debloater scripts found on the internet can cause instability issues since there are changes being made that otherwise are normally not. You should reach out to whoever created that application. Sent from my iPhoneOn Dec 14, 2022, at 2:54 AM, katastrophe92 ***@***.***> wrote:﻿

**Comment by katastrophe92** [comment]
> I’m not familiar with the SophiApp. What app is that? 


I think you now about what app I'm taking about. Anyway this is the response i got 


   - How does it know in the first place and why does it cares?

Hi. Because SophiApp checks if there is temp folder in system drive that is a leftover from Sycnex' script. And due to I care about users in general, and my user, I think that I have to inform them that they have broken their system by using such scripts which were made by know-nothing dev.

  - Are this 2 apps are not compatible, fighting each other or the developers of this apps do not each other?

Technically they're compatible. We do not fight against anyone. As I wrote before my duty as a specialist inform users what they've done to their systems. Just look at Sycnex's script last commit. Guess something? Read his readme? HE even doesn't care at all about any possible harm that can be made to users' OS. Look at opened issues. Oh, just 262 unreplied. That's so nice. A guy who knows how to write 3 string in PowerShell made a script and it became widespread across YouTube, and nobody cares about the code quality, the responsibility for any changes made by his script. He doesn't care at all! We care. Fixing every reported bug. Because we're specialists in this scope. We are paid salaries at our corporations for our knowledge. And want to keep people safe from apps like this.

  - Where are still files or trails of Windows10Debloater on my PC?

Remove just this (or toggle the switch in SophiApp): %SystemDrive%\Temp\Windows10Debloater

If you're wondering about Win 10 Tweaker checks, this is very famous tweaker contains a backdoor with the ability to run any malicious code from the dev's cloud. I wrote the article explaining how it works. But nobody cares. The dev keeps a botnet of poor users.

P.S. Also if you have another warnings, it means that your Windows is damaged. Sycnex's checks is one of many. :) Feel free to ask any question. Also if you prefer Telegram: we have a group to discuss: https://t.me/sophia_chat
